environments/prod/prod.tfvars

```diff
@@ -606,6 +606,7 @@ frontends = [
         "942380",
         "942400",
         "942430",
+        "932100",
```

Comment:
please sort these in order, and this needs a SNOW change approved by IA

Reply:
Done